### PR TITLE
fix: wrong image name in release manifests

### PR DESCRIPTION
**Pull request by Xunzhuo**
image

Fixed:

image

image tag of EG in release manifests is incorrect 😢, let us fix it.

Signed-off-by: Xunzhuo <bitliu@tencent.com>

### CODE REVIEW

File: tools/make/common.mk [hunk 1]
```diff
@@ -33,7 +33,7 @@ endif
 # Set the version number. you should not need to do this
 # for the majority of scenarios.
 ifeq ($(origin VERSION), undefined)
-VERSION := $(shell git describe --abbrev=0 --dirty --always --tags | sed 's/-/./g')
+VERSION := $(shell git describe --abbrev=0 --dirty --always --tags)
```
Comment (arkodg) [inline tools/make/common.mk:36]: isn't TAG tied to REV ?
https://github.com/envoyproxy/gateway/blob/15859d119a4e73d452585b2f164e0256b7c11f08/tools/make/image.mk#L15

Comment (Xunzhuo) [reply tools/make/common.mk:36]: REV=$(shell git rev-parse --short HEAD)

REV is the latest commit not tag.

Comment (arkodg) [reply tools/make/common.mk:36]: and the release GHA is overriding the tag value here https://github.com/envoyproxy/gateway/blob/cc44487f4037b87a14f45597be5485ab3899d9b9/.github/workflows/release.yaml#L46

Comment (Xunzhuo) [reply tools/make/common.mk:36]: Right, we can just switch to TAG.

Comment (arkodg) [reply tools/make/common.mk:36]: can we just delete VERSION for now, since its not used anywhere and we are basing versioning off TAG

Comment (Xunzhuo) [reply tools/make/common.mk]: Yes, deleted.

**Review by arkodg — APPROVED** [review]
thanks for addressing this @Xunzhuo !

**Pull request merged**